component/common/loki/utils/convert.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import "github.com/prometheus/common/model"
+
+// Convert a map of strings to a prometheus LabelSet.
```

Comment:
The go doc convention is to start the comment with the name of the function: https://go.dev/blog/godoc
```suggestion
// ToLabelSet converts a map of strings to a prometheus LabelSet.
```